core/include/detray/geometry/volume.hpp

```diff
@@ -29,44 +30,53 @@ class volume {
     // used for sfinae
     using volume_def = volume<object_registry, range_type, array_type>;
 
+    /** Default constructor**/
+    volume() = default;
```

Comment:
Is a default constructed volume well-defined?

Reply:
Should be okay. All members have default value